files/chef-marketplace-cookbooks/chef-marketplace/templates/default/reckoner.rb.erb

```diff
@@ -18,3 +23,21 @@ db.password = '<%= node['chef-compliance']['postgresql']['db_superuser_password'
 aws.product_code = '<%= node['chef-marketplace']['reckoner']['product_code'] %>'
 aws.usage_dimension = '<%= node['chef-marketplace']['reckoner']['usage_dimension'] %>'
 <% end %>
+
+# phone home configurations
+<% if node['chef-marketplace']['platform'] == 'gce' %>
+phone_home.enabled true
+<% else %>
+phone_home.enabled false
+<% end %>
+phone_home.endpoint 'https://0pg896lyqi.execute-api.us-east-1.amazonaws.com/test/eetee'
```

Comment:
I had it hard-coded as MVP since this will likely not be the final resting place (currently in the chef-aws account, probably not where it will be in production). I'll make it an attribute regardless though.